pages/doc/2019_46.x_release_notes.md

```diff
@@ -83,3 +83,7 @@ A <strong>node map</strong> shows colored 3-D cubes ordered from highest to lowe
 * We now allow lower case or all caps for the strings `AND`, `OR`, and `NOT` for better legibility of long combinations of metrics.
 * The [`timespan() event management function`](event_timespan.html) now supports adding tags with a time expression. Before, only Epoch time was supported.
 * The terms `high` and `low` are now reserved keywords. If you want to query for the string "high" or "low" you have to use double quotes. For example, `ts(my.metric AND current_value="high")` queries for `my.metric` but returns only instances where the point tag `current_value` is set to "high".
+
+## Navigate to the Tracing UI from the Service Metrics Dashboard
```

Comment:
Maybe 
Navigate to Tracing UI from Service Metrics Dashboard 
or
Drilldown from Service Metrics Dashboard to Tracing UI
(it's a heading)

Reply:
That sounds a lot better. Thank you!